README.md

```diff
@@ -10,6 +10,9 @@ comments.  For more information about OpenShift, visit https://openshift.redhat.
 or the OpenShift support page
 https://openshift.redhat.com/support.
 
+RHC is typically built on Ruby 1.8.7.  If you are running into any dependency 
+issues when using RHC with later versions of Ruby, please try building the RHC 
+gem using a later version of Ruby to get the correct dependencies.
```

Comment:
I think you meant "try building the RHC gem using a**n earlier** version of Ruby to get the correct dependencies"